src/utils/normalization.ts

```diff
@@ -52,24 +52,38 @@ export const normalize = (payload: OperationsRepository) => {
 
       const deployments = Object.fromEntries(
         Object.entries(api.deployments).map(([key, value]) => {
-          return [
-            key,
-            Object.fromEntries(
-              Object.entries(value).map(([key, value]) => {
-                if (key === 'secrets') {
-                  const envBuffer = Buffer.from((value as Secrets).content);
-                  const content = Object.entries(parse(envBuffer))
-                    .map(([key, _value]) => key)
-                    .concat([''])
-                    .join('=""\n');
-
-                  return [key, { ...value, content }];
-                }
-
-                return [key, value];
-              })
-            ),
-          ];
+          const airnode = Object.fromEntries(
+            Object.entries(value.airnode).map(([key, value]) => {
```

Comment:
any idea why this doesn't work well with typescript ? 
```
      const deployments = Object.fromEntries(
        Object.entries(api.deployments).map(([key, value]) => {
          return [
            key,
            Object.fromEntries(
              Object.entries(value).map(([deploymentName, deployment]) => {
                return [
                  deploymentName,
                  Object.entries(deployment).map(([key, value]) => {
                    if (key === 'secrets') {
                      const envBuffer = Buffer.from((value as Secrets).content);
                      const content = Object.entries(parse(envBuffer))
                        .map(([key, _value]) => key)
                        .concat([''])
                        .join('=""\n');

                      return [key, { ...value, content }];
                    }

                    return [key, value];
                  }),
                ];
              })
            ),
          ];
        })
      );
```
This is what I tried first but typescript wasn't happy. This was the outputted type on my IDE
```
[k: string]: {
        [k: string]: (string | {
            ois: {
                version: string;
                endpoints: {
                    description?: string | undefined;
                    externalDocs?: string | undefined;
                    summary?: string | undefined;
                    ... 4 more ...;
                    reservedParameters: {
                        ...;
                    }[];
                }[];
                oisFormat: string;
                title: string;
                apiSpecifications: {
                    ...;
                };
            }[];
            chains: {
                ...;
            }[];
            nodeSettings: {
                ...;
            };
            triggers: {
                ...;
            };
            apiCredentials: {
                ...;
            }[];
        } | {
            filename: string;
            content: string;
        } | {
            ...;
        })[][];
    };
```

and this is what it is expecting 
```
const deployments: Record<string, {
    airnode: {
        aws?: {
            filename: string;
            content: string;
        } | undefined;
        config: {
            ois: {
                version: string;
                endpoints: {
                    description?: string | undefined;
                    externalDocs?: string | undefined;
                    ... 5 more ...;
                    reservedParameters: {
                        ...;
                    }[];
                }[];
                oisFormat: string;
                title: string;
                apiSpecifications: {
                    ...;
                };
            }[];
            chains: {
                ...;
            }[];
            nodeSettings: {
                ...;
            };
            triggers: {
                ...;
            };
            apiCredentials: {
                ...;
            }[];
        };
        secrets: {
            filename: string;
            content: string;
        };
    };
    airkeeper: {
        ...;
    };
}>
```

For some reason that code is not treating the airnode/airkeeper keys as fixed strings and instead as random strings

Reply:
I'm looking now.
I don't mind if you break these assignments up for readability, in fact I'd encourage it.

Reply:
Yeah, I'm not sure where exactly the issue is coming from. I know it's related to the processing of airkeeper and airnode types. There are some other urgent matters requiring attention, so for now you can set `          ) as z.infer<typeof airkeeperDeploymentSchema>;` at the end of the airnode and airkeeper processor maps (line ~71 and ~87 in normalization.ts). This won't break our validation, so should all work nicely. This may surface additional bugs when you run `normalize-and-validate`.

Reply:
This doesn't break validation either, so Ill go ahead and merge it if your fine with it